packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,107 @@
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_cusd_circulating
+11353464.550486518
+```
+
+#### CP-DOTO 
+
+For API endpoints useful for listing that follow [CMC requirements](https://docs.google.com/document/d/1S4urpzUnO2t7DmS_1dc4EL4tgnnbTObPYXvDeBnukCg/edit#)
+
+##### Summary
+Summary overview.
+
+```sh 
+$ curl https://thecelo.com/api/v0.1.js?method=ex_summary
+
+{"trading_pairs":"CELO_CUSD","last_price":2.6143,"lowest_ask":2.5933609958506225,"highest_bid":2.5676,"base_volume":37524.32000000003,"quote_volume":14714.520000000002,"price_change_percent_24h":3.7027120070382127,"highest_price_24h":2.649,"lowest_price_24h":2.4787}}
+```
+
+##### Assets
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_assets
+
+{"code":"200","msg":"success","data":{"CELO":{"name":"CELO","unified_cryptoasset_id":"5567","can_withdraw":"true","can_deposit":"true","min_withdraw":"0.000000000000000001","max_withdraw":"0.000000000000000001","maker_fee":"0.00","taker_fee":"0.005"},"CUSD":{"name":"Celo Dollars","unified_cryptoasset_id":"825","can_withdraw":"true","can_deposit":"true","min_withdraw":"0.000000000000000001","max_withdraw":"0.000000000000000001","maker_fee":"0.00","taker_fee":"0.005"}}}
+```
+
+##### Ticker
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_ticker
+
+{"code":"200","msg":"success","data":{"CELO_CUSD":{"base_id":"5567","quote_id":"825","last_price":2.6124,"quote_volume":14789.520000000002,"base_volume":37720.30000000003,"isFrozen":"0"}}}
+```
+
+##### Orderbook
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_orderbook
+
+{"code":"200","msg":"success","data":{"timestamp":1601061465962,"bids":[["2.5964","100"]],"asks":[["2.622606871230003","100"]]}}
+```
+
+##### CELO cUSD
+```sh 
+$ curl https://thecelo.com/api/v0.1.js?method=ex_celocusd
+
+{"code":"200","msg":"success","data":{"CELO_CUSD":[{"trade_id":2697341,"timestamp":1601061491,"price":0.38238291620515147,"quote_volume":25,"base_volume":65.37948987916423,"type":"Sell"},{"trade_id":2697336,"timestamp":1601061466,"price":0.382293821845672,"quote_volume":25,"base_volume":65.39472670341044,"type":"Sell"}]}}
+```
+
+### Whitepapers
+To learn about the Celo Protocol, please refer to the [whitepaper](https://celo.org/papers/Celo_A_Multi_Asset_Cryptographic_Protocol_for_Decentralized_Social_Payments.pdf).
+
+If you need more information to explore other aspects of the Celo Protocol, there’s a [useful links](https://docs.celo.org/#useful-links) page.
+
+To learn more about the Stability Mechanism, you can find it over [here](https://docs.celo.org/celo-codebase/protocol/stability). 
+The [Stability Analysis Whitepaper](https://celo.org/papers/Celo_Stability_Analysis.pdf) and [blog post](https://medium.com/celohq/a-look-at-the-celo-stability-analysis-white-paper-part-1-23edd5ef8b5) are also linked.
+
+If you want to find more information about the Celo Reserve, please visit the link [here](https://celoreserve.org ).
```

Comment:
Maybe add a description about the reserve and what the information provided is. 

Consider leaving the link as is (w/out word "here") since it's an easy and memorable URL